example_scoring.py

```diff
@@ -0,0 +1,17 @@
+from mw.api import Session
+
+from revscoring.extractors import APIExtractor
+from revscoring.languages import english
+from revscoring.scorers import MLScorerModel
+
+api_session = Session("https://en.wikipedia.org/w/api.php")
+extractor = APIExtractor(api_session, english)
+
+filename = "models/reverts.halfak_mix.trained.model"
```

Comment:
I think that is fair and we should actually have this example added at some point.  Right now, I don't think we have a good place to link to.  And I expect that most of our users won't be building their own models, so this is a more important example to have now.